daml-script/test/daml/TestExceptions.daml

```diff
@@ -0,0 +1,50 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module TestExceptions where
```

Comment:
Could you also add a test that throw an exception during evaluation (e.g. via `throwPure`) and catches it? I'd like to make sure we catch exceptions that are thrown during evaluation.

Similarly it might be nice to have a test that exhibits an exception during the evaluation & during the execution of an exception handler.

Reply:
Added a bunch of test cases in e1dbd1643ffa29a6511289daa84e47274cbcf5ce. Do those cover the cases you were thinking off?

Reply:
merging but happy to add more test cases if I missed something

Reply:
👍  Thank you, that's perfect!